pkg/model/stack_serializer.go

```diff
@@ -1138,7 +1139,16 @@ func (v *StackVolume) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		return err
 	}
 
-	parts := strings.SplitN(raw, ":", 2)
+	parts := strings.Split(raw, ":")
+	if runtime.GOOS == "windows" {
+		if len(parts) >= 3 {
```

Comment:
I think what you've done covers 99.99% of cases, but to bear in mind there could also be:
- `len(parts) == 4` : this should be 2 absolute windows paths separated by a `:`. Example `C:\somewhere\local:C:\somewhere\in\the\container`
- `len(parts) == 3` : we don't know if the absolute path is in the host or in the container. We could see if Windows allows drive names other than letters, and then evaluate a solution from there

Reply:
- len(parts) == 4 is not supported. Right now we only support linux machines
- len(parts) == 3 as it's not supported on the container we can always infer that is on the host